transform_to_tf/src/library/transform_to_tf.cpp

```diff
@@ -0,0 +1,51 @@
+#include "transform_to_tf/transform_to_tf.hpp"
+
+namespace transform_to_tf {
+
+TransformToTf::TransformToTf(const ros::NodeHandle& nh,
+                             const ros::NodeHandle& nh_private)
+    : nh_(nh),
+      nh_private_(nh_private),
+      verbose_(kDefaultVerbose),
+      local_frame_name_(kDefaultLocalFrameName),
+      global_frame_name_(kDefaultGlobalFrameName) {
+  // Getting data and params
+  subscribeToTopics();
+  advertiseTopics();
+  getParametersFromRos();
+}
+
+void TransformToTf::subscribeToTopics() {
+  // Subscribing to the required data topics
+  transform_sub_ = nh_.subscribe("transform_to_be_converted", 1,
+                                 &TransformToTf::transformCallback, this);
+}
+
+void TransformToTf::advertiseTopics() {
+  // Advertising topics
+  tf_timer_ = nh_.createTimer(ros::Duration(0.01),
+                              &TransformToTf::publishTFTransform, this);
+}
+
+void TransformToTf::getParametersFromRos() {
+  // Retrieving the parameters
+  nh_private_.getParam("verbose", verbose_);
```

Comment:
I prefer the use of .param over getParam, as while it works I don't think that getParam is specified to not modify the variable when it hasn't found a value in the parameter server.